services/webhook/tests/test_llm_client.py

```
@@ -379,6 +379,25 @@ def capture(url, *, json, headers, timeout):
     assert captured[0]["timeout"] == lc._REVIEW_TIMEOUT_SECONDS
 
 
+def test_cave_calls_are_tagged_interactive_priority() -> None:
+    """Grug's own review + elder-fallback calls carry X-Spark-Priority:
+    interactive so the spark-gateway priority queue (infra#1763-1767 ADR
+    follow-up) lets them jump ahead of Hermes's long agentic turns on the
+    same shared, single-generation-slot Ollama target - the exact 2026-07-12
+    incident this header exists to prevent."""
+    captured: list = []
+
+    def capture(url, *, json, headers, timeout):
```

Ruff (0.15.20)

[warning] Missing return type annotation for private function `capture`

(ANN202)